start.js

```diff
@@ -1,31 +1,32 @@
 const config = require('./config')
 const log = require('./app/services/log')
-var throng = require('throng')
+var CronJob = require('cron').CronJob
 var processTasks = require('./app/process-tasks')
 
+var asyncWorkerCron = config.ASYNC_WORKER_CRON
 var numberOfWorkers = config.ASYNC_WORKER_CONCURRENCY
-var frequency = config.ASYNC_WORKER_FREQUENCY
 var startWeb = config.ASYNC_START_WEB === 'true'
-throng({ workers: numberOfWorkers }, startWorker)
 
-function startWorker (id) {
-  log.info(`Started worker ${id}`)
+for (var i = 0; i < numberOfWorkers; i++) {
+  var workerId = i + 1
```

Comment:
Forget the number of workers, that was a throng concept. If we want multiple instances of the worker we can start multiple processes in PM2